.github/workflows/image-scan.yml

```
@@ -102,109 +136,146 @@ jobs:
-        _hln() {
-          if [ ${1} -gt 0 ]; then
-            echo "**${1}** :x:"
-          else
-            echo "${1} :heavy_check_mark:"
-          fi
-        }
-
-        for jsonFile in $lwResultFiles; do
-          image=$(jq -r '.image' $jsonFile)
-          cveC=$(jq -r '.stats.Critical.total' $jsonFile)
-          cveH=$(jq -r '.stats.High.total' $jsonFile)
-          cveM=$(jq -r '.stats.Medium.total' $jsonFile)
-          cveL=$(jq -r '.stats.Low.total' $jsonFile)
-          cveI=$(jq -r '.stats.Info.total' $jsonFile)
-          comment="${comment}\n| ${image} | $(_hln $cveC) | $(_hln $cveH) | $cveM | $cveL | $cveI |"
-        done
-
-        runUrl="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}#user-content-scan-results"
-        comment="${comment}\n\n<sup>[Go to job summary for more details...]($runUrl)</sup>"
-
-        # Set multiline output.
-        echo 'comment<<EOF' >> $GITHUB_OUTPUT
-        echo -e "$comment" >> $GITHUB_OUTPUT
-        echo 'EOF' >> $GITHUB_OUTPUT
-
-    - name: Add PR comment (if content changed)
-      if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize')
-      uses: actions/github-script@v6
-      with:
-        retries: 3
-        retry-exempt-status-codes: 400,401
-        script: |
-          function removeDynamicValues(body) {
-            // remove links and dynamic image tag hashes
-            return body
-               .replace(/\]\(http[^)]+\)/gm, "](#)")
-               .replace(/(docker\.io\/[^ ]+)-\w{7}/gm, "$1-#######")
           }
+          lwResultFiles="$(find . -type f -name '*.lacework-result.json' | sort)"
+          aquaResultFiles="$(find . -type f -name '*.aqua-result.json' | sort)"
+          echo "## <a name=\"scan-results\"></a>Lacework scan results" >> $GITHUB_STEP_SUMMARY
+          summaryForScanner "$lwResultFiles" "lw"
+          echo "## <a name=\"scan-results\"></a>Aqua scan results" >> $GITHUB_STEP_SUMMARY
+          summaryForScanner "$aquaResultFiles" "aqua"
 
-          // Get the existing comments.
-          const {data: comments} = await github.rest.issues.listComments({
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            issue_number: context.payload.number,
-          })
-
-          // Find last bot comment.
-          const lastBotComment = comments.slice().reverse().find(comment => comment.user.login === 'github-actions[bot]')
+      - name: Build PR comment
+        if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize')
+        id: build-pr-comment
+        shell: bash
+        run: |
+          createPRCommentTable() {
+            comment="${comment}\n| Image | Critical | High | Medium | Low | Info |"
+            comment="${comment}\n|---|---|---|---|---|---|"
+            
+            for resultJSONFile in $aquaResultFiles; do
```

Rado, have you also tested these changes? :smile: 
```suggestion
            for resultJSONFile in $1; do
```

---

Originally, yes, not the new ones, that's why I converted it to a draft =D testing atm in a private repo.

---

Last minute changes were a bit careless from me, I admit :sweat:

---

Ok, don't worry, we are almost there :smile: